packages/plugin-eslint/src/lib/runner/index.ts

```diff
@@ -0,0 +1,58 @@
+import type { Audit, AuditOutput, RunnerConfig } from '@code-pushup/models';
+import { toArray } from '@code-pushup/utils';
+import { mkdir, writeFile } from 'fs/promises';
+import { dirname } from 'path';
+import { lint } from './lint';
+import { lintResultsToAudits } from './transform';
+
+export const RUNNER_OUTPUT_PATH =
+  'node_modules/.code-pushup/eslint/runner-output.json';
```

Comment:
Why do we use this folder in `node_modules`

Reply:
The idea is to use some hidden directory, so we don't pollute the Git repo. Since basically all our users will already have a `node_modules` folder in `.gitignore`, it's a convenient place to put these files. I got the idea from other packages which use it:

![image](https://github.com/flowup/quality-metrics-cli/assets/34691111/b2d667cd-7dd9-4b03-a8cb-61857d0220b3)

An alternative approach that's also very common is to create a dedicated folder in the root (`.code-pushup` in our case) and automatically add it to `.gitignore` in some init script (e.g. `.angular`, `.firebase`). I think that's also a viable alternative, although it adds an extra setup step.